src/main/java/org/jabref/logic/importer/fetcher/CompositeSearchBasedFetcher.java

```diff
@@ -17,22 +21,33 @@
 
 public class CompositeSearchBasedFetcher implements SearchBasedFetcher {
 
-    public static final String FETCHER_NAME = "SearchAll";
+    public static final String FETCHER_NAME = "SearchSelected";
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CompositeSearchBasedFetcher.class);
 
-    private final Set<SearchBasedFetcher> fetchers;
+    private static Set<SearchBasedFetcher> fetchers;
```

Comment:
Why is this `static`? It is initialized in the constructor. Thus, it should belong to the instance. Just remove `static`